qa-tests-backend/src/test/groovy/DefaultPoliciesTest.groovy

```diff
@@ -112,6 +113,11 @@ class DefaultPoliciesTest extends BaseSpecification {
             .setImage("us.gcr.io/acs-san-stackroxci/qa-multi-arch:nginx-1.12")
             .addLabel ( "app", "test" )
             .setCommand(["sleep", "600"]),
+        new Deployment()
+            .setName(UNSIGNED_REDHAT)
+            .setImage("registry.redhat.io/redhat/community-operator-index:v4.19")
```

Comment:
This test will fail when this image disappears. At that point, probably all Red Hat images will be signed. Is this something we need to worry about now? And if so, what can be done about it?

Perhaps just checking that the policy exists is enough - the code that handles checking the images for violations is already tested at a lower level.

Reply:
The nature of this policy requires a "real" red hat image hosted at `registry.redhat.io` or similar, which we don't control, so this uncertainty will always be there. Most likely it will be fine, as plenty of customer environments depend on these images. If the image disappears at some point, let's decide then how to handle this.